aip/general/0157.md

```diff
@@ -14,28 +14,27 @@ needs to give the user control over which fields it sends back.
 
 ## Guidance
 
-APIs **may** support partial responses in one of the three ways:
+APIs **may** support partial responses in one of two ways:
 
-### Field masks parameter
+### Field masks parameter (recommended)
```

Comment:
I'm not a big fan of adding this `(recommended)` because this is how the side nav and hash portion of the url are formed. Rather, I'd advocate for saying that they are **recommended** in text, below.

Reply:
Yeah, I'm not in love either...

Reply:
Removed